src/cryptonote_core/service_node_quorum_cop.cpp

```diff
@@ -197,7 +194,7 @@ namespace service_nodes
             break;
 
           m_obligations_height = std::max(m_obligations_height, start_voting_from_height);
-          for (; m_obligations_height < (height - REORG_SAFETY_BUFFER_IN_BLOCKS); m_obligations_height++)
+          for (; m_obligations_height < (height - MAX_REORG_BLOCKS); m_obligations_height++)
```

Comment:
I like the "safety buffer" name much better.  The "max reorg" name implies that this is somehow a limit, but as far as I can see it isn't (and mustn't be!): it is merely the target maximum implied by a checkpointing mechanism working in normal times.  It is still possible for checkpoints to be missed and for this maximum to be exceeded; it's a rare case that shouldn't normally happen, but it *is* something we need to handle (even if suboptimally).

Reply:
You're right,  I forgot about missed checkpoints leading to > the usual number of blocks reorganisable.